src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -93,8 +112,14 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     projectNumber: changeRequest.wbsNum.projectNumber,
     workPackageNumber: 0
   });
+  const [reviewerIds, setReviewerIds] = useState<number[]>([]);
```

Comment:
hmmmm, do we want to use a form for this instead? @RChandler234

Reply:
just saw this, yeah we should

Reply:
Oh I see. 

What's the idea for us to use useForm hook instead of useState? Saw other examples of useForm usage, and it seems like we retrieve 
```
handleSubmit,
    control,
    formState: { isValid },
    reset
```
fields from useForm hook. I want to ask you guys how we use these fields in our style instead of reviewerIds and setReviewerrIds. 

Thanks!

Reply:
useForm allows you to set up a form, giving you more control over validating inputs, displaying error messages for fields in the form, settings values in the form, and controling when/ how submitting the form happens. (basically just gives you more control than using a useState).

However, now that I'm looking through the code again, useState is probable fine here, because we're never gonna add another type of field to this form, so setting up a whole useForm might be overcomplicating it, so don't worry ab changing that for now